src/index.html

```diff
@@ -8,9 +8,42 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="images/Search.svg">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      class="form"
+      action="#"
+      method="get"
+    >
+      <img class="dandruff" src="images/Search.svg" alt="search">
+      <label for="search"></label>
+      <input
+        data-qa="keypress"
+        name="search"
+        id="search"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search"
+      >
+    </form>
+    <form
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
    </form>
    
    <form
```